package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua

```diff
@@ -223,9 +223,7 @@ local function mimedecode_message_body(src, msg, filecb)
 			until not spos
 
 			if found then
-				-- We found at least some boundary. Save
-				-- the unparsed remaining data for the
-				-- next chunk.
+				-- We found at least some boundary.
```

Comment:
The comment is still correct, there is nothing to change here.

Reply:
But where is the remaining data used after that line? the if clause ends and else is another case.

Reply:
lchunk is used again in the next call of snk. Unlike data, lchunk is an upvalue.

Reply:
AH, ow I get it: the comment was misleading, I thought, with the remaining "data" they mean the variable "data", but the comment is not about the variable but just uses the word "data"

maybe we should rephrase this then?

> save the unparsed remaining data-part for the next chunk.

Reply:
changed to "the remaining string"

Reply:
Even if it refered to the "variable" data, the comment would still be accurate: The remaining (variable) "data" is saved (by assigning it to a different variable). Please just leave this comment unchanged.

Reply:
ok, done ;)